aea/protocols/generator.py

```diff
@@ -161,56 +168,71 @@ def _message_class_str(self) -> str:
         cls_str += '    def __init__(self, message_id: int, target: int, performative: str, contents: Dict, **kwargs):\n'
         cls_str += '        \"\"\"Initialise.\"\"\"\n'
         cls_str += '        super().__init__(message_id=message_id, target=target, performative=performative, contents=contents, **kwargs)\n\n'
-
-        # variables
-        # cls_str += str.format('        self.performatives = {}\n', self._performatives_set())
-        cls_str += str.format('        self.speech_acts = {}\n\n', self._speech_acts_str())
+        cls_str += str.format('        self.speech_acts_definition = {}\n\n', self._speech_acts_str())
         cls_str += '        assert self.check_consistency()\n\n'
 
+        # Properties
         cls_str += '    @property\n'
-        cls_str += '    def performatives(self) -> set:\n'
+        cls_str += '    def performatives_definition(self) -> set:\n'
```

Comment:
Please relabel this to `allowed_performatives` or `permitted_performatives` as the variable name is confusing and the docstring clear.

Reply:
It should also be a private `_var_name` variable and not a property since it is only used by the message object internally

Reply:
Made it a classmethod but kept it public so could be used by others.

Reply:
I don't think using a method to expose a class variable is ideal.